nixos/tests/services/webhook.nix

```diff
@@ -0,0 +1,28 @@
+{ pkgs, ... }:
+{
+  name = "webhook";
+
+  nodes = {
+    webhookMachine = { pkgs, ... }: {
+      services.webhook = {
+        enable = true;
+      };
+    };
```

Comment:
Maybe add a client node to test the `openFirewall` option?

Reply:
I was wondering why forwarding the port wasn't working... the manual doesn't mention how to communicate between nodes or if they're in the same network. I'll try and figure that out, but if it's not straight forward, a port-forward it'll be!

Reply:
After searching for a while, I couldn't find documentation on communication between VM and went for the simple option of forwarding port a port on the host to the guest after adding the `openFirewall` option. To use a client machine, I'd need a code snippet to show how that's done.

Reply:
Oh, what I was trying to say was to put another node in the `nodes` attribute; two VMs talking to each other.
That's how most networked tests are written.
I don't really know how port forwarding compares to having two VMs in a network.